NitroxClient/GameLogic/RemotePlayer.cs

```diff
@@ -123,7 +111,7 @@ public void SetPilotingChair(PilotingChair newPilotingChair)
                     mpCyclops.Exit();
                 }
 
-                RigidBody.isKinematic = AnimationController["cyclops_steering"] = (newPilotingChair != null);
+                RigidBody.isKinematic = AnimationController["cyclops_steering"] = newPilotingChair != null;
```

Comment:
minor: can we leave parenthesis around statements as-is?  This statement is already hard to read.